### PR TITLE
add floats to java rewrite engine

### PR DESCRIPTION
**Pull request by dwightguth**
@andreistefanescu please review.

### CODE REVIEW

File: tests/regression/float/hs_err_pid23530.log [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+
+
+
+Memory: 4k page, physical 32638464k(10262932k free), swap 8074236k(8074236k free)
+
+vm_info: OpenJDK 64-Bit Server VM (24.51-b03) for linux-amd64 JRE (1.7.0_55-b14), built on Apr 25 2014 13:32:38 by "buildd" with gcc 4.6.3
+
+time: Wed May 21 14:45:39 2014
+elapsed time: 1 seconds
+
```
Comment (radumereuta) [inline tests/regression/float/hs_err_pid23530.log]: Should this be committed?

Comment (dwightguth) [inline tests/regression/float/hs_err_pid23530.log]: oops, no, thank you

File: tests/regression/float/hs_err_pid24899.log [hunk 2]
```diff
@@ -0,0 +1,715 @@
...
+
+
+
+Memory: 4k page, physical 32638464k(10295940k free), swap 8074236k(8074236k free)
+
+vm_info: OpenJDK 64-Bit Server VM (24.51-b03) for linux-amd64 JRE (1.7.0_55-b14), built on Apr 25 2014 13:32:38 by "buildd" with gcc 4.6.3
+
+time: Wed May 21 16:04:56 2014
+elapsed time: 1 seconds
+
```
Comment (radumereuta) [inline tests/regression/float/hs_err_pid24899.log]: Same question for this one too.

**Comment by dwightguth** [comment]
I am closing this. I will merge this into the unified-builtins2 branch, because I don't particularly want to have to resolve the snarling mess of problems involving logik.

**Pull request closed without merging**